config/merge.go

```diff
@@ -54,6 +62,14 @@ func Merge(existingServices *ServiceConfigs, environmentLookup EnvironmentLookup
 
 	adjustValues(serviceConfigs)
 
+	if options.Postprocess != nil {
+		var err error
+		serviceConfigs, err = options.Postprocess(serviceConfigs)
```

Comment:
Our use case for preprocess is to convert booleans to strings in `environment` and `labels` to maintain backwards compatibility with Compose files that haven't been updated. No immediate use case for postprocess, but it seemed like something that may be handy in the near future. Adding those fields may also be helpful in the future, so I'm definitely open to it!